modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/graphql_mutation.ftl

```diff
@@ -76,16 +84,16 @@ public class Mutation {
 					_${freeMarkerTool.getSchemaVarName(javaMethodSignature.schemaName)}ResourceComponentServiceObjects, this::_populateResourceContext,
 					${freeMarkerTool.getSchemaVarName(javaMethodSignature.schemaName)}Resource -> {
 
-						Page paginationPage = ${freeMarkerTool.getSchemaVarName(javaMethodSignature.schemaName)}Resource.${javaMethodSignature.methodName}(${arguments});
+						Page paginationPage = ${freeMarkerTool.getSchemaVarName(javaMethodSignature.schemaName)}Resource.<#if hasMultipartBody>${javaMethodSignature.methodName}Multipart<#else>${javaMethodSignature.methodName}</#if>(${arguments});
```

Comment:
As we are including the if clause several times, I am thinking about a cleaner way to do this. For example, having JavaMethodSignature and GraphQLJavaMethodSignature with different names?